2_sars-cov-2.md

```diff
@@ -4,9 +4,38 @@ title: SARS-CoV-2
 permalink: /sars-cov-2/
 ---
 
-V-pipe is part of the [SIB resources supporting SARS-CoV-2 research](https://www.sib.swiss/about-sib/news/10660-sib-resources-supporting-sars-cov-2-research).
+[![SARS-CoV-2]({{ "/img/sars-cov-2_header.png" | relative_url }})](https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2)
+
+
+We have released a new version of V-pipe specifically adapted  to analyze high-throughput sequencing data of the SARS-CoV-2 virus.
+
+ - available in the [sars-cov2 branch](https://github.com/cbg-ethz/V-pipe/tree/sars-cov2) of the Github repository
+ - default reference sequence is [NC_045512](https://www.ncbi.nlm.nih.gov/nuccore/NC_045512), identical to GenBank entry [MN908947](https://www.ncbi.nlm.nih.gov/nuccore/MN908947)
+ - [default configuration](https://github.com/cbg-ethz/V-pipe/blob/sars-cov2/vpipe.config) file assumes SARS-CoV-2 NGS data
+ - new version [ShoRAH 2](https://github.com/cbg-ethz/shorah/releases) integrated for SNPs calling
```

Comment:
s/SNPs/SNVs/

Reply:
Thanks, this one did slip thourgh the last round of fixes.